assets/src/dashboard/components/card-grid-item/index.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types'; // import styled from 'styled-components';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+const StyledGridItem = styled.div`
+  width: ${({ theme }) => theme.grid.desktop.itemWidth};
+  height: ${({ theme }) => theme.grid.desktop.itemHeight};
+  background-color: #d9dbdd;
```

Comment:
this is temporary and will be removed as card item is fleshed out separately

Reply:
One last thing, let's make this a "placeholder" background color in the theme.